loadtest/user/userentity/actions.go

```diff
@@ -32,11 +31,17 @@ func (ue *UserEntity) Login() error {
 		return err
 	}
 
-	_, resp := ue.client.Login(user.Email, user.Password)
+	loggedUser, resp := ue.client.Login(user.Email, user.Password)
 	if resp.Error != nil {
 		return resp.Error
 	}
 
+	// We need to set user again because the user ID does not get set
+	// if a user is already signed up.
+	if err := ue.store.SetUser(loggedUser); err != nil {
+		return err
+	}
+
```

Comment:
I think this was the missing part to make the tests *resumable* without clearing the db every time.

Reply:
Yep